ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/metrics/DeltaFilesMetricReporter.java

```diff
@@ -212,6 +213,22 @@ public static void mergeDeltaFilesStats(AcidDirectory dir, long checkThresholdIn
         }
       }
     }
+
```

Comment:
It would be better to separate this block out into a new method, for better readability